Assets/HoloToolkit/Input/Scripts/InputSources/InputManagerExtensions.cs

```diff
@@ -0,0 +1,314 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    public static partial class InputManagerExtensions
```

Comment:
Why make this a extension class? I understand that it removes this code form InputManager itself, but I'm not a big fan of making extension classes when we actually have access to the original class' code. Extension classes make sense when you want to extend a class that is provided to you through a DLL or something, but in this case, I think this code should be added to InputManager itself.

I'm also not a fan of really not a fan of partial classes. Partial classes are great to handle generated code, but they should never be used as an extension mechanism IMO.